exercises/sheet4/sheet4.tex

```diff
@@ -74,11 +110,79 @@
   \]
   for any polytope $P$, where $p_0 = \frac{1}{\vol(P)}\int_P x\,\text{d}x$ denotes the barycenter of $P$.
   \bigskip\bigskip
-\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}, possibly referring to \cite{bs-1992}.
+\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}.
   
 \end{enumerate}
 
-\bibliographystyle{plain}
-\bibliography{sheet4}
+
+
+
+\begin{enumerate}
+
+\item It is not possible, since we cannot obtain $Q^c$ (or $\mathcal L^\downarrow(Q^c)$) just from $Q$, $\pi$ and $c$. \newline For example, take $q = 1$, $Q$ any 1-polytope (any interval), $\pi(x,y) = x$ and $c = (0,1)$, then $\pi^c$ is the identity in $P$, so $Q^c = P$, but we are not given $P$, so it's impossible to get $Q^c$ or just $\mathcal L^\downarrow(Q^c)$.
+
+\item Directly from the linearity of $f$ and the linearity property of the integral:
+
+\begin{equation*}
+	\vol(P) f(p_0)
+	= \vol(P) f\left( \frac1{\vol (P)}\int_P x\,\mathrm{d}x \right)
+	= f\left(\int_P x\,\mathrm{d}x \right) 
+	= \int_P f(x) \,\mathrm{d}x
+\end{equation*}
+
+\item I will enumerate the parts of the sketch of Ziegler's proof which are not clear and I'll explain them:
+
+\begin{enumerate}[(i)]
+
+\item $\Sigma(P,Q)$ is convex.
+
+\begin{proof}
+Given two points $a,b \in \Sigma(P,Q)$ and their corresponding sections $\gamma_a, \gamma_b$. Then $(ta + (1-t)b) = \int t\gamma_a + (1-t)\gamma_b = \int \gamma dx$ since the convex combination of 2 sections is a section (by the linearity of the integral and $\pi$).
+\end{proof}
+
+\item $\dim(\Sigma(P,Q)) \leq p-q$ with $\dim(P) = p$, $\dim(Q) = q$.
+
+\begin{proof}
+We have to see that $\pi^{(-1)}(r_0)$ has dimension p-q. Observing that, since we have supposed that the dimensions of $P$ and $Q$ are maximal, $\ker(\pi) = p - q$ and so we get the result.
+\end{proof}
+
+
+
+\item "Every piecewise linear section that is not tight can be changed locally in two directions; thus it can be written as a convex combination of two other sections".
+
+\begin{proof}
+
+\end{proof}
+
+
+\item "To detect the vertices of $\Sigma(P, Q)$, we use that they arise as the unique maxima for generic linear functions $c \in (\mathbb{R}^p)^*$. However, if $c$ is generic, then
+every fiber $\pi^{-1}(r)$ for $ r \in Q$ has a unique maximal element with respect to $c$".
+
+\begin{proof}
+The problem of this part is the definition of $generic$ linear functions. In this case there's only one possibility for the definition of generic:
+
+\begin{definition}
+A linear function $c$ will be generic with respect to $\Gamma(P,Q)$ if $c$ is generic with respect to $P$ and with respect to $\pi$, that is that $c$ is not parallel to $\pi^{(-1)}(r)$ for $r \in Q$.
```

Comment:
"will be" -> "is"
Is it "\Gamma" or "\gamma"? Who is $\Gamma$? Do you mean "$\Sigma"?
What do the two phrases "$c$ is generic wrt $P$" and $c$ is generic wrt $\pi$" mean?